phpBB/phpbb/profilefields/manager.php

```diff
@@ -479,4 +495,161 @@ public function build_insert_sql_array($cp_data)
 
 		return $cp_data;
 	}
+
+	/**
+	* Disable all profile fields of a certain type
+	*
+	* This should be called when an extension which has profile field types
+	* is disabled so that all those profile fields are hidden and do not
+	* cause errors
+	*
+	* @param string $profilefield_type_name Type identifier of the profile fields
+	*/
+	public function disable_profilefields($profilefield_type_name)
+	{
+		// Get list of profile fields affected by this operation, if any
+		$pfs = array();
```

Comment:
Please use more meaningful variable names, this is not C. Something like `$profile_fields` should suffice.